buildout.cfg

```diff
@@ -53,7 +53,7 @@ webtest = git https://github.com/Pylons/webtest.git
 WSGIProxy2 = git https://github.com/lrowe/WSGIProxy2.git
 zope.sqlalchemy = git https://github.com/zopefoundation/zope.sqlalchemy.git
 pytest-bdd = git https://github.com/lrowe/pytest-bdd.git branch=allow-any-step-order
-snovault = git https://github.com/ENCODE-DCC/snovault.git rev=1.0.11
+snovault = git https://github.com/ENCODE-DCC/snovault.git branch=snovault-refactor-low-lvl
```

Comment:
This change shouldn't be merged right @caseylitton?

Reply:
I tell people to do it this way because for whatever reason it shows up as a merge conflict.  So it actually can't be merged as is.  We revert the commit after finished with QA and code review.